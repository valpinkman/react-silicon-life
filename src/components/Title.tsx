import * as React from 'react'

interface TitleProps {
  children: React.ReactNode;
}

const Title: React.FunctionComponent<TitleProps> = ({ children }: TitleProps) => <h1>{children}</h1>

export default Title
