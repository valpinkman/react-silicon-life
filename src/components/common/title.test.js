import React from 'react'
import renderer from 'react-test-renderer'
import Title from './title'

test('should display the correct text', () => {
  const content = 'This is a title'

  const component = renderer.create(<Title>{content}</Title>)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
