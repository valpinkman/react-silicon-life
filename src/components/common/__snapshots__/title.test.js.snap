// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the correct text 1`] = `
Array [
  <h1>
    This is a title
  </h1>,
  <style
    jsx={true}
  >
    
      h1 {
        font-family: sans-serif;
      }
    
  </style>,
]
`;
